packages/x-date-pickers/src/AdapterDateFns/AdapterDateFns.test.tsx

```diff
@@ -26,6 +26,7 @@ describe('<AdapterDateFns />', () => {
       const adapter = new AdapterDateFns({ locale: enUS });
       const date = adapter.date(TEST_DATE_ISO_STRING)!;
 
+      // TODO@v7: can be removed after v7 release
```

Comment:
I'm using `TODO v7`
We can reconsider the best comment later, but I think we need to stick to one for this major to be able to get all of them with a single search :+1: